src/main/java/com/ibreath/controller/UserController.java

```diff
@@ -0,0 +1,38 @@
+package com.ibreath.controller;
+
+import com.ibreath.resource.dto.UserDto;
+import com.ibreath.service.UserService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+
+@RestController
+@Api("Measure")
+public class UserController {
+
+    private UserService service;
+
+    @ApiOperation("Get measure Data")
+    @RequestMapping(value="/user/{userId}", method = RequestMethod.GET)
```

Comment:
I think this path should be better /users/etc.. I just add the S to user. This change is also working for the other paths